recipe-client-addon/bootstrap.js

```diff
@@ -10,14 +10,18 @@ XPCOMUtils.defineLazyModuleGetter(this, "LogManager",
   "resource://shield-recipe-client/lib/LogManager.jsm");
 XPCOMUtils.defineLazyModuleGetter(this, "ShieldRecipeClient",
   "resource://shield-recipe-client/lib/ShieldRecipeClient.jsm");
+XPCOMUtils.defineLazyModuleGetter(this, "AboutStudiesProtocol",
+  "resource://shield-recipe-client/lib/AboutStudiesProtocol.jsm");
 
 this.install = function() {};
 
 this.startup = async function() {
   await ShieldRecipeClient.startup();
+  AboutStudiesProtocol.register();
```

Comment:
We shouldn't be modifying `bootstrap.js` if we can avoid it. We should treat `ShieldRecipeClient` as the entrypoint for anything in the add-on. This is because it is very hard to control when bootstrap runs in tests, and putting everything in another file makes that a lot easier.